### DISCUSSION TITLE
What are the main ingredients in the increasingly popular "BLAT" sandwich?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#484] [Galaxy Trivia] [answered]
*Answer using emojis*

**Comment by peterbaccaro** [comment]
:bread: :bacon: :green_salad: :avocado: :tomato:

The main ingredients in the "BLAT" sandwich are the bread for the sandwich, bacon, lettuce, avocado and tomato!

**Reply by ParanoidUser** [reply]
Thank you for your enthusiasm, Peter! 🤓 As you may have noticed, there are "bonus" questions and a "puzzle" addition to the trivia. I'd love to hear your feedback on this. Any advice on how to make it more interesting and entertaining is greatly appreciated!

**Reply by peterbaccaro** [reply]
I think the variety of questions and answers is good.  The puzzle questions make you think and having to answer using #RRGGBB or with emojis can make it a little more challenging.   How about some anagram style questions?  Or guess the county based on a description of the flag and/or some #RRGGBB colors?  Or guess the celebrity or person based on some clues?  Or guess the recipe based in some emojis of ingredients?

**Reply by ParanoidUser** [reply]
Great ideas! Thank you for helping us to improve. More questions to come! 👍

**Comment by mayankjaiswal1709** [comment]
The BLAT sandwich is a variation of the classic BLT sandwich that includes avocado in addition to bacon, lettuce, and tomato. So the main ingredients in a BLAT sandwich are:

Bacon: crispy strips of bacon
Lettuce: usually crispy iceberg lettuce, but any leafy green can be used
Avocado: ripe avocado slices or guacamole spread
Tomato: fresh, ripe tomato slices
Some variations of the BLAT sandwich may also include additional ingredients such as mayonnaise, cheese, or a fried egg.

**Comment by thuongtruong1009** [comment, marked as answer]
The best BLAT sandwich recipe features luscious layers of bacon, lettuce, avocado and tomato! Ranch balsamic mayonnaise adds extra flavor. Each type of bread has different flavor and texture profiles that will elevate your sandwich to World Class status! Some of our favorites include:

Sourdough bread
Whole wheat bread
Multigrain bread
White bread
Potato bread
Challah
Brioche
